namespaces/exec.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/docker/libcontainer/system"
 )
 
+const (
+	EXIT_SIGNAL_OFFSET = 128
```

Comment:
Because that is how all the other consts are for signals and most of the consts in libcontainer